react/ContactsList/data.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "_id": "013a6414-9727-4a4e-8b35-d3b9946a17c6",
+    "_type": "io.cozy.contacts",
+    "email": [
+      {
+        "address": "isabelle.durand@cozycloud.cc",
+        "primary": true
+      }
+    ]
+  },
```

Comment:
Why do we need this? I don't mind it but it makes the review a bit harder, especially the snapshots.

Reply:
I wanted to add a contact with no fullname or name in the data, so we have this case in the styleguide.